libs/core/src/helpers/flag-theme-color.type.ts

```diff
@@ -0,0 +1 @@
+export type FlagThemeColor = 'success' | 'warning' | 'danger' | 'semi-light' | 'transparent';
```

Comment:
Consider adding this type in the flag component file, this is how most other types are implemented when they dont need to be shared across components.